generator/lib/src/realm_field_info.dart

```diff
@@ -80,7 +80,8 @@ class RealmFieldInfo {
   Iterable<String> toCode() sync* {
     final getTypeName = type.isRealmCollection ? basicMappedTypeName : basicNonNullableMappedTypeName;
     yield '@override';
-    yield "$mappedTypeName get $name => RealmObjectBase.get<$getTypeName>(this, '$realmName') as $mappedTypeName;";
+    final validateBacklink = isRealmBacklink ? ", validateBacklink: true" : "";
+    yield "$mappedTypeName get $name => RealmObjectBase.get<$getTypeName>(this, '$realmName'$validateBacklink) as $mappedTypeName;";
```

Comment:
This could check if the object isManaged and throw here.

Reply:
The purpose was the check and the exception to be in one place. Otherwise this will be repeated for all the backlink properties:
```dart
RealmResults<Source> get oneToMany {
    if (!isManaged) {
      throw RealmException("Using backlinks is only possible for managed(persisted) objects.");
    }
    return RealmObjectBase.get<Source>(this, 'oneToMany') as RealmResults<Source>;
  }
```

Reply:
why would be a problem to be repeated? This is generated code.

Reply:
Ok, Done.